README.md

```diff
@@ -223,27 +227,28 @@ service studentData on studentServiceListener {
         // Table is converted to JSON.
         var jsonConversionValue = json.convert(dataTable);
         if (jsonConversionValue is error) {
-            status = { "Status": "Data Not available" };
+            log:printError("Error in converting the data table from database to JSON", err = jsonConversionValue);
         } else {
             status = jsonConversionValue;
         }
         // Sending back the converted JSON data to the request made to this service.
         response.setJsonPayload(untaint status);
         var result = caller->respond(response);
         if (result is error) {
-            log:printError("Error sending response", err = result);
+            log:printError("Error in sending response", err = result);
         }
 
         if (childSpanId is int) {
             _ = observe:finishSpan(childSpanId);
         } else {
             log:printError("Error attaching span ", err = childSpanId);
         }
-        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default ootb system span.
+        // The below function adds tags that are to be passed as metrics in the traces. These tags are added to the default system span.
```

Comment:
The below function adds tags to be passed as metrics in the traces. These tags are added to the default system span.